config.yaml.sample

```diff
@@ -1,23 +1,50 @@
 postgresql:
   host: localhost
   port: 5432
+  # multiple databases can be specified, and these are iterated
+  # through to balance query loads. This is useful when connecting to
+  # pgbouncer, which can dispatch to different back end databases
+  # based on the name.
   dbnames: [osm]
   user: osm
   password:
 queries:
   config: ../vector-datasource/queries.yaml
   template-path: ../vector-datasource/queries
+  # whether to reload the jinja query templates. This should be off in
+  # production.
   reload-templates: true
+
+# format extensions to support
```

Comment:
We should mention buffered MVT files are an option in the comment (but we don't support them in prod).